src/components/Sing/SideBar/TrackItem.vue

```diff
@@ -0,0 +1,373 @@
+);
+const shouldPlayTrack = computed(() =>
+  getOrThrow(shouldPlayTracks(store.state.tracks), props.trackId),
+);
+
+const setTrackPan = (pan: number) => {
+  if (store.state.songUndoableTrackOptions.panAndGain) {
+    store.dispatch("COMMAND_SET_TRACK_PAN", { trackId: props.trackId, pan });
+  } else {
+    store.dispatch("SET_TRACK_PAN", { trackId: props.trackId, pan });
+  }
+};
+
+const setTrackGain = (gain: number) => {
+  if (store.state.songUndoableTrackOptions.panAndGain) {
+    store.dispatch("COMMAND_SET_TRACK_GAIN", {
+      trackId: props.trackId,
+      gain,
+    });
+  } else {
+    store.dispatch("SET_TRACK_GAIN", { trackId: props.trackId, gain });
+  }
+};
+
+watch(
+  () => track.value.name,
+  () => {
+    temporaryTrackName.value = track.value.name;
+  },
+);
+
+const temporaryTrackName = ref(track.value.name);
+
+const updateTrackName = () => {
+  if (temporaryTrackName.value === track.value.name) return;
+  if (temporaryTrackName.value === "") {
+    temporaryTrackName.value = track.value.name;
+    return;
+  }
+  setTrackName(temporaryTrackName.value);
+};
+
+const setTrackName = (name: string) => {
+  store.dispatch("COMMAND_SET_TRACK_NAME", { trackId: props.trackId, name });
+};
+
+const setTrackMute = (mute: boolean) => {
+  if (store.state.songUndoableTrackOptions.soloAndMute) {
+    store.dispatch("COMMAND_SET_TRACK_MUTE", { trackId: props.trackId, mute });
+  } else {
+    store.dispatch("SET_TRACK_MUTE", { trackId: props.trackId, mute });
+  }
+};
+
+const setTrackSolo = (solo: boolean) => {
+  if (store.state.songUndoableTrackOptions.soloAndMute) {
+    store.dispatch("COMMAND_SET_TRACK_SOLO", { trackId: props.trackId, solo });
+  } else {
+    store.dispatch("SET_TRACK_SOLO", { trackId: props.trackId, solo });
+  }
+};
+
+const selectedTrackId = computed(() => store.state.selectedTrackId);
+
+const trackCharacter = computed<
+  { character: CharacterInfo; style: StyleInfo } | undefined
+>(() => {
+  if (!track.value.singer) return undefined;
+
+  for (const character of store.state.characterInfos[
+    track.value.singer.engineId
+  ]) {
+    for (const style of character.metas.styles) {
+      if (style.styleId === track.value.singer.styleId) {
+        return { character, style };
+      }
+    }
+  }
+  return undefined;
+});
+const selectTrack = () => {
+  store.dispatch("SET_SELECTED_TRACK", { trackId: props.trackId });
+};
+
+const deleteTrack = () => {
+  store.dispatch("COMMAND_DELETE_TRACK", { trackId: props.trackId });
+};
+
+const singerName = computed(() => {
+  const character = trackCharacter.value;
+  if (!character) return "（不明なキャラクター）";
+  return character.character.metas.speakerName;
+});
+</script>
+<style scoped lang="scss">
+@use "@/styles/colors" as colors;
+@use "@/styles/variables" as vars;
+
+.track-detail-container {
+  padding: 0;
+
+  border-bottom: 1px solid colors.$sequencer-sub-divider;
+
+  // draggingクラス：SideBarのDraggableにより追加/削除される。
```

Comment:
（細かいけど）

「：」はいろんな意味に取られうる（「は」「とは」「比率」）ので、一意な表現のが良いかも。
```suggestion
  // draggingクラスはSideBarのDraggableにより追加/削除される。
```